### PR TITLE
created artist component, trying to grab featured image

### PR DESCRIPTION
**Pull request by feeohnah**

### CODE REVIEW

File: frontend/pages/artists.js [hunk 1]
```diff
@@ -1,19 +1,41 @@
...
+  static async getInitialProps() {
+    const res = await fetch(endpoint);
+    const data = await res.json();
+    return {data};
+  }
 
-class ArtistIndexPage extends Component {
   render() {
+    const artists = this.props.data;
+    console.log(
```
Comment (feeohnah) [inline frontend/pages/artists.js]: This works, I can log out the exact image url I need, but then when I try to put that into the {image} prop below it says that ['wp:featuredmedia'] is undefined, wtf

File: frontend/components/Image/Image.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React, {Component} from 'react';
+import {StyledImage} from './styles';
+
+function Image({src, alt, title}) {
+  return <StyledImage scr={src} alt={alt} title={title} />;
```
Comment (cartogram) [inline frontend/components/Image/Image.js]: title?

File: frontend/components/Slider.js [hunk 3]
```diff
@@ -1,37 +1,37 @@
-import React, { Component } from "react";
-import fetch from "isomorphic-unfetch";
-import { Config } from "../config";
-import Link from "next/link";
+import React, {Component} from 'react';
+import fetch from 'isomorphic-unfetch';
+import {Config} from '../config';
+import Link from 'next/link';
 
 export default class Slider extends Component {
```
Comment (cartogram) [inline frontend/components/Slider.js]: make this a dumb component

File: frontend/pages/artists.js [hunk 4]
```diff
@@ -1,19 +1,36 @@
...
+    const artistlistMarkup = artists.map((artist) => {
+      const image =
+        artist._embedded['wp:featuredmedia'][0].media_details.sizes.large
+          .source_url;
+      return (
+        <li key={artist.id}>
+          <Artist
+            url={artist.link}
+            title={artist.title.rendered}
+            artistimage={image}
```
Comment (cartogram) [inline frontend/pages/artists.js]: camelCase

File: frontend/pages/category.js [hunk 5]
```diff
@@ -43,10 +42,10 @@ class Category extends Component {
       );
     });
     return (
-      <Menu menu={this.props.headerMenu} />
-      <h1>{this.props.categories[0].name} Posts</h1>
       <div>
-        {postsMarkup}
+        <Menu menu={this.props.headerMenu} />
```
Comment (cartogram) [inline frontend/pages/category.js]: Save yourself some typing and start destructuring these. 
```
const {headerMenu, categories} = this.props;
```

File: frontend/components/Slider.js (hunk 3 above)
Comment (feeohnah) [reply frontend/components/Slider.js]: Yeah, haven't touched this yet

File: frontend/pages/category.js (hunk 5 above)
Comment (feeohnah) [reply frontend/pages/category.js]: where do I put that?

File: frontend/pages/category.js [hunk 6]
```diff
@@ -1,29 +1,28 @@
...
       );
       const posts = await postsRes.json();
-      return { categories, posts };
+      return {categories, posts};
     }
-    return { categories };
+    return {categories};
   }
   render() {
     if (this.props.categories.length == 0) return <Error statusCode={404} />;
```
[the remark this answers is not in the document]
Comment (feeohnah) [reply frontend/pages/category.js]: I haven't really edited this file yet, not sure how what you're asking for is different to what I have

File: frontend/pages/post.js [hunk 7]
```diff
@@ -3,7 +3,6 @@ import fetch from 'isomorphic-unfetch';
 import Error from 'next/error';
 import {Config} from '../config.js';
 import withLayout from '../decorators/withLayout';
-import Main from '../components/Main';
 
 class Post extends Component {
```
[the remark this answers is not in the document]
Comment (feeohnah) [reply frontend/pages/post.js]: ? I'm not sure how/where to put that

File: frontend/styles/links.js [hunk 8]
```diff
@@ -0,0 +1,50 @@
+import {css} from 'styled-components';
+
+const linkMain = css`
```
[the remark this answers is not in the document]
Comment (feeohnah) [reply frontend/styles/links.js]: I think that's overkill for this

File: frontend/styles/global.js [hunk 9]
```diff
@@ -27,4 +27,8 @@ injectGlobal`
     color: ${theme.LinkPrimary.CurrentColor};
     text-decoration: none;
   }
+  h1 {
+    ${(props) => props.theme.pxScale[3]};
```
[the remark this answers is not in the document]
Comment (feeohnah) [reply frontend/styles/global.js]: making the h1 a specific size

**Pull request merged**